nilmtk/dataset_converters/ampds/convert_ampds.py

```diff
@@ -22,15 +22,15 @@
                      'S': ('power', 'apparent'),
                      'St': ('energy', 'apparent')}
 
-TIMESTAMP_COLUMN_NAME = "TS"
+TIMESTAMP_COLUMN_NAME = "unix_ts"
 TIMEZONE = "America/Vancouver"
 
 def convert_ampds(input_path, output_filename, format='HDF'):
     """
-    Convert AMPds R2013 as seen on Dataverse. Download the files
+    Convert AMPds2 R2016 as seen on Dataverse. Download the files
```

Comment:
It might be worth to add a comment here to clarify how to download those, e.g. https://github.com/nilmtk/nilmtk/pull/850#issuecomment-755602278